vg_chart_generator/vg_chart_generator/MANIFEST.in

```diff
@@ -0,0 +1 @@
+include vg_chart_generator/data/Video_Games_Sales_as_at_22_Dec_2016.csv
```

Comment:
This is the file that we have to write to make sure that when packaging everything together that this file is included with the distribution package. Right?